lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3500,6 +3500,7 @@ namespace {
       SmallVector<FuncDecl *, 4> methods;
       SmallVector<ConstructorDecl *, 4> ctors;
       SmallVector<TypeDecl *, 4> nestedTypes;
+      SmallPtrSet<Decl *, 16> allMembers;
```

Comment:
Optional: thoughts on adding a comment here that says this set's only purpose is to ensure we don't add the same member twice?

Reply:
Done